tests/models/levit/test_modeling_levit.py

```diff
@@ -335,6 +335,10 @@ def test_training_gradient_checkpointing(self):
             loss.backward()
 
     def test_problem_types(self):
+
+        if not (is_torch_less_than_1_9 or is_torch_greater_or_equal_than_1_10):
+            self.skipTest(reason="This test fails with PyTorch 1.9: some CUDA issue")
```

Comment:
You could just test that the version is equal to 1.9.0, it would result in a simpler test!

Reply:
Hi! The original past CI was running with torch 1.9.1. But I also checked with 1.9.0, doesn't work neither.
Would you be OK with the following:
```python
parsed_torch_version_base = version.parse(version.parse(torch.__version__).base_version)
if parsed_torch_version_base.base_version.startswith("1.9"):
    skip ....
```

Reply:
Sure!